packages/builder/tsconfig.json

```diff
@@ -1,9 +1,6 @@
 {
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
-    "composite": true,
```

Comment:
This is not required, and it was causing vscode to complain. This tsconfig was added just because of the paths, as it's not a ts project